app/assets/stylesheets/styles/post-edit.scss

```diff
@@ -10,12 +10,12 @@ label {
   margin: 0;
 }
 
-#container {
+#edit-container {
```

Comment:
This would make more sense as a class.  I know you probably only have one, but that's not really a good reason.  ID's should be used on things that are unique.  An edit-container?  Not so much. The main navbar?  I guess... but. me personally, I wouldn't.  Also this would look silly in HTML.  Now I have to do `<div id="edit-container"></div>`, which suddenly looks terrible when I want to add additional classes `<div id="edit-container" class="hoverable dark-themed"></div>`.  Now think down the road when you suddenly want two for whatever reason.  You're going to have to go back and change this anyway.  Also you can only have one ID so what's the point.

Recap

- When in doubt, use a class
- Think you should use an ID? Prolly not
- I should never see more than a couple ID's used a page
- ID should be used for unique stuff and used sparingly
- Classes should be used for everything because it makes sense.  You can derive from multiple classes
- "But my Javascript selects by ID!!!" Select the root node for your editor by ID, and then grab everything else by class

Reply:
This was given the class editContainer and the id was removed.

Reply:
I suppose the last thing I'm wondering is what the filenames mean, and is it inconsistent to have the files name be named-with-dashes or camelCased now that the CSS has camel case?

Reply:
My original intent of those filenames was to indicate the controller and view the styles were used on. I know that filename isn't really standard so I renamed the files like below

styles/post/edit.scss
styles/post/list.scss

if you have any other ideas for stylesheet filenames lets tackle that in issue #21